tests/http/locales.test.js

```diff
@@ -0,0 +1,115 @@
+var should = require('should');
+var paths = require('../../data/paths.js');
+var server = require('../../server');
+var instance = server({ useDomains: false });
+
+var type = 'text/html; charset=utf-8';
+var cache = 'max-age=300, must-revalidate, public';
+
+describe('paths with locale', () => {
+  paths.forEach(function(page) {
+    it(`${page.pathname}`, (done) => {
+      instance.inject({
+        url: page.pathname
+      }, (response) => {
+        should(response.statusCode).equal(200);
+        should(response.headers['content-type']).equal(type);
+        should(response.headers['cache-control']).equal(cache);
+        done();
+      });
+    });
+  });
+});
+
+describe('paths with locale', () => {
```

Comment:
I think I can remove these because above sets pretty much covers everything, but I just thought I want hardcoded ones in case the array above didn't work somehow?